lib/routes/account.js

```diff
@@ -204,7 +204,8 @@ module.exports = function (
             email: account.email,
             emailCode: account.emailCode,
             emailVerified: account.emailVerified,
-            verifierSetAt: account.verifierSetAt
+            verifierSetAt: account.verifierSetAt,
+            createdAt: parseInt(request.query.createdAt)
```

Comment:
We should be explicit about that happens if `request.query.createdAt` is not specified; IIUC this would currently produce `NaN` and we depend on the token-creation code to correctly interpret that as meaning "use the current time".